.goreleaser.yml

```diff
@@ -29,12 +29,26 @@ builds:
 
 archives:
   - format: tar.gz
+    name_template: >-
+      {{ .Binary }}_{{ .Version }}_
+      {{- if eq .Os "darwin" }}osx{{ else }}{{ .Os }}{{ end }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}x86_32
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- with .Arm }}v{{ . }}{{ end }}{{ with .Mips }}_{{ . }}{{ end }}{{ if not (eq .Amd64 "v1") }}{{ .Amd64 }}{{ end }}
```

Comment:
Can you unpack this a little?  Is there some default value that you're overriding, which changed in between the goreleaser versions?

Reply:
Or is this just the new way to do `replacements`?

Reply:
That's the new way of doing `replacements`, and I took the default name_template for binaries, while updating it with the required if/else to make it work exactly like it was before